rag_experiment_accelerator/config/config.py

```diff
@@ -12,6 +12,7 @@
 from rag_experiment_accelerator.embedding.factory import EmbeddingModelFactory
 from rag_experiment_accelerator.llm.prompts import main_prompt_instruction
 from rag_experiment_accelerator.utils.logging import get_logger
+from rag_experiment_accelerator.config.credentials import _get_env_var
```

Comment:
Why are you calling this internal function directly, rather than using one of the existing classes?

`MAX_WORKER_THREADS` isn't a credentials thing, so why not pull this directly from the environment?

Reply:
you are correct, there' no relation to credentials, fixed